micro-rdk/src/common/ota.rs

```diff
@@ -189,20 +192,38 @@ impl OtaService {
             );
             uri = hyper::Uri::from_parts(parts).map_err(|e| OtaError::Other(e.to_string()))?;
         };
-        let io = self
-            .connector
-            .connect_to(&uri)
-            .map_err(|e| OtaError::Other(e.to_string()))?
-            .await
-            .map_err(|e| OtaError::Other(e.to_string()))?;
 
-        let (mut sender, conn) = {
-            http2::Builder::new(self.exec.clone())
-                .max_frame_size(16_384) // lowest configurable value
-                .timer(H2Timer)
-                .handshake(io)
-                .await
-                .map_err(|e| OtaError::Other(e.to_string()))?
+        let (mut sender, conn) = loop {
+            match self.connector.connect_to(&uri) {
+                Ok(acceptor) => {
```

Comment:
Why `acceptor` here, when you are doing a `connect`?

Reply:
you're right. it should be `connector` according to the [return value](https://github.com/viamrobotics/micro-rdk/blob/0fedde4460f0f88510ad4fe14877521ffae30e20/micro-rdk/src/esp32/tcp.rs#L480). I got it mixed up with the method above it, `accept_connection`.